xml/obs_scm_bridge.xml

```diff
@@ -12,31 +12,34 @@
          xmlns:ns5="http://www.w3.org/2000/svg"
          xmlns:ns4="http://www.w3.org/1999/xhtml"
          xmlns:ns="http://docbook.org/ns/docbook">
-  <title>SCM Bridge [EXPERIMENTAL]</title>
+  <title>SCM Bridge</title>
 
   <sect1 xml:id="sec.obs.obs_scm_bridge">
     <title>SCM Bridge</title>
 
     <sect2 xml:id="sec.obs.obs_scm_bridge.setup.introduction">
       <title>Introduction</title>
 
-      <para>The SCM bridge allows to setup single packages or entire projects in any trusted SCM repositories. 
+      <para>The SCM bridge allows to setup single package or entire project sources to be stored and
+            maintained in any trusted SCM repositories. 
             However, the only supported SCM is currently git.</para>
 
-      <para>This allows to manage the entire package sources in an external SCM repository. While the classic
-      OBS services for scms (like obs_scm or tar_scm) are using the workflows inside of OBS, these workflows
-      have to happen inside of the SCM provider instance. OBS will just verify the authenticity of that resource in
-      future.</para>
+      <para>This allows to run the entire packaging source update in an external SCM repository using the
+      native tooling. It is recommended to have this SCM server on the same trust level as the OBS
```

Comment:
The first sentence of this paragraph is potentially unclear in two ways.

First, it refers to "the entire packaging source update" - the definite article ("the") indicates that there is some definite (specific) packaging source update that is being referred to, but I'm not sure this is the case? What do you mean by "the entire packaging source update", here, exactly?

Second, it says "using the native tooling". Again, the definite article is used, but I'm not sure the reader knows which "native tooling" you mean here?

Reply:
tried to be more clear now.